src/rabbitalerter.js

```diff
@@ -0,0 +1,208 @@
+/**
+ * An alerting system for message queues.
+ * @module rabbitalerter
+ */
+const assert = require('assert');
+const taskcluster = require('taskcluster-client');
+
+/**
+ * An interface used to generate various TaskCluster Notifications.
+ *
+ * @class RabbitAlerter
+ */
+class RabbitAlerter {
+  /**
+   * @param {number} config.alerter.messageCountTolerance       - When a queue holds more messages
+   *                                                              than this tolerance, an alert will be created.
+   * @param {number} config.alerter.messagePublishRateTolerance - When the number of messages published per second
```

Comment:
Note: This assumes that `message_stats.publish` is actually the absolute number of messages published since the queue was created. I hope that is the case, and I hope that is present for all queues, we should have a look, maybe it's only high traffic queues that has this... Or maybe it's only queues with active consumers.
(I guess if it requires active consumers we can alert if `message_stats` it's not present 30 min).

Also we would have to store the absolute, number so that if the number goes down (because the queue was deleted and recreated) then we just discard whatever history we have and start over.